### DISCUSSION TITLE
Not able to login using github!

### DISCUSSION THREAD

**Discussion by AhmadIbrahiim** [vercel/community#542] [Help] [answered]
Hello Vercles 🚀 

I used to log in to my vercel account: https://vercel.com/aibrahim using Github! but now I'm not able to do so!

Anyone can help me with that?

**Comment by swarnava** [comment, marked as answer]
What is the error you are seeing?

**Reply by AhmadIbrahiim** [reply]
@swarnava ```There is no Vercel account linked with this GitHub account. [Sign up?](https://vercel.com/signup?next=%252Faibrahim)```

**Reply by swarnava** [reply]
That sounds like the case then. Either you have signed up via email address or you had likely used different GitHub account. Do you remember your vercel email or domain which is registered with us? Please raise a support request using https://vercel.com/support/request so we can triage this.

**Reply by AhmadIbrahiim** [reply]
Yeah, I remember the domain, but I have only one Github account!

domain is : ahmed-ibrahim.com

**Reply by AhmadIbrahiim** [reply]
Sadly, support/request asks me to login :/

**Reply by swarnava** [reply]
Your account is linked with ahmed@04****t.com. You can use the same to login. Since GitHub isn't linked with your account, it won't work.

**Reply by AhmadIbrahiim** [reply]
Oh got it. Thank you so much! It was set to a very old email!! all is good now. Thank you
